source/lac/scalapack.cc

```diff
@@ -546,16 +547,15 @@ ScaLAPACKMatrix<NumberType>::eigenpairs_symmetric(const bool compute_eigenvector
        * for jobz==N only eigenvalues are computed, for jobz='V' also the eigenvectors of the matrix are computed
        */
       char jobz = compute_eigenvectors ? 'V' : 'N';
-      char range;
+      char range='Z';
```

Comment:
I would put 'A' as default for range. Or is there a specific reason for choosing 'Z'?

Reply:
no, it's just something for it to be initialized in case it's forgotten to be so down there